### ISSUE TITLE
In MyBagActivity if no internet case

### ISSUE THREAD

**Issue by midhilaj-flutter-developer** [wakralabconsulting/qmecomandroid#205]
In MyBagActivity if no internet it is not showing any message

**Comment by midhilaj-flutter-developer** [comment]
```
 @Override
    protected void onResume() {
        super.onResume();
        if(util==null)util=new Util();
        if(util.isNetworkAvailable(getApplicationContext()))
        {
            myBagPresenter.setup();
        }else {
           nointernetDialog= util.showRetryDialog(this, new View.OnClickListener() {
                @Override
                public void onClick(View v) {

                    nointernetDialog.cancel();
                    onResume();
                }
            });
            nointernetDialog.show();
        }
    }
```

**Comment by midhilaj-flutter-developer** [comment]
solved 74691bf3f38a908e283673fac87c00a6578b98d3
